spec/adapters/googlecharts_spec.rb

```diff
@@ -41,43 +41,48 @@
       ['2013'],
     ]
   end
+  let(:user_options) {{
+    listeners: {
+      select: "alert('A table row was selected');"
+    }
+  }}
   let(:query_string) {'SELECT A, H, O, Q, R, U LIMIT 5 OFFSET 8'}
   let(:data_spreadsheet) {'https://docs.google.com/spreadsheets/d/1XWJLkAwch'\
               '5GXAt_7zOFDcg8Wm8Xv29_8PWuuW15qmAE/gviz/tq?gid=0&headers='\
               '1&tq=' << query_string}
   let (:plot_spreadsheet) {
     Daru::View::Plot.new(
       data_spreadsheet,
-      {type: :column, width: 800}
+      {type: :column, width: 800},
+      user_options
     )
   }
   let (:table_spreadsheet) {
     Daru::View::Table.new(
       data_spreadsheet, {width: 800}
     )
   }
-  let(:data_table) {Daru::View::Table.new(data)}
-  let(:user_options) {{chart_class: 'Chartwrapper'}}
```

Comment:
It would be better if you write new variables for new specs, rather than modifying existing variables completely.